### DISCUSSION TITLE
integrazione pergola bioclimatica con HA..possibile?

### DISCUSSION THREAD

**Discussion by reclus23** [indomus/forum#2790] [GENERALE - Discussioni generiche]
ciao ho da poco, ho fatto installare  una pergola bioclimatica (della BT group) che è collegata all'impianto elettrico sia per l'illuminazione che per la regolazione del tetto e delle tende. Il tutto viene gestito da un telecomando.
Secondo voi c'è la possibilità attraverso l'installazione (di non so cosa) a portare il tutto in HA?
grazie

**Comment by Bergoglio** [comment]
Aiutami a capire, perché non ho mai avuto modo di vederne una dal vivo.
Arriva l'alimentazione elettrica alla pergola dentro una centrale unica da cui partono le alimentazioni alle luci, motori delle tende etc. ? Ed il telecomando agisce su questa centrale ?
Oppure arriva l'alimentazione elettrica che viene smistata a luci e motori delle tende, poi ognuno di questi ha una mini centralina sua di gestione e solo il telecomando è unico ma agisce sulle diverse centraline ?

**Comment by reclus23** [comment]
ciao grazie per l'intervento. la centrale è unica e da qui parte tutto, luci, tende etc. 
Il telecomando agisce sulla centrale.
grazie

**Comment by Bergoglio** [comment]
La soluzione più semplice è sostituire il telecomando con quei componenti tipo i Broadlink.
Qui su Indomus ci sono diversi articoli e prove in materia.

**Comment by reclus23** [comment]
bergoglio grazie mille, non conoscevo i broadlink...ora studierò! grazie

**Reply by Bergoglio** [reply]
Non c'è di che !

**Reply by KC-inDomus** [reply]
https://indomus.it/formazione/come-usare-gli-attuatori-broadlink-su-home-assistant/

**Reply by reclus23** [reply]
KC grazie anche a te. dopo la segnalazione di Bergoglio sto leggendo un pò ed ho ordinato RM4 pro!

**Comment by reclus23** [comment]
broadlink arrivato e l'ho subito testato con la prima tenda. tutto funziona perfettamente (grazie ancora). ora voglio integrare la tenda in HA e soltanto dopo che avrò capito tutto per bene, passerò alla bioclimatica (visto che ha più tasti e funzioni).
Per HA ho preso spunto dal link segnalato da KC ed ho deciso di provare a strutturare la tenda come una vera e proprio unità strutturata, attraverso smartIR, anche se leggendo la guida, le tende, non sarebbero proprio contemplate....

**Comment by reclus23** [comment]
ciao a tutti non c'è verso di associare il telecomando della bioclimatica con il broadlink...per l'associazione sto provando per il momento con l'app di broadlink (ad HA tenterò dopo).  ho provato sia selezionando IR che RF ma nulla....
questa è la foto del telecomando... avete idee? grazie

https://tinypic.host/image/WhatsApp-Image-2023-12-30-at-14.25.50.Q0CTz


in pratica con i due pulsanti tondi in alto si seleziona la parte della bioclimatica su cui si vuole agire (per es una delle tre tende, luci, o alette del tetto). Per fare l'associazione seleziono prima per es. tenda uno e poi cerco di collegare al broadlink il pulsante della salita, poi, quello della discesa e quello dello stop..nulla...

**Reply by Bergoglio** [reply]
Non è che è un telecomando rolling code ?
Cosa dice il manuale ?
Se è un telecomando IR dovrebbe non eseguire nessun comando quando non è puntato verso la centralina (od il ricevitore, se separato)

**Comment by crug80** [comment]
Hai provato a chiedere al produttore se c'è una qualche predisposizione verso sistemi  di controllo esterni? 
Prima di smanettare, io cercherei una soluzione ufficiale..

**Comment by reclus23** [comment]
> Non è che è un telecomando rolling code ? Cosa dice il manuale ? Se è un telecomando IR dovrebbe non eseguire nessun comando quando non è puntato verso la centralina (od il ricevitore, se separato)

temo anche io che sia rolling code. il bello è che non mi hanno lasciato alcun libretto d'istruzione e le informazioni che sto raggruppando le sto prendendo dalla parte tecnica del sito..(cose che capitano quando non si è personalmente presenti al montaggio e si delega........)
i telecomandi (della tenda esterna e della bioclimatica) non sono IR.. per la tenda esterna, infatti il broadlink riesce ad inviare l'impulso da dentro casa stando almeno a 10 mt di distanza dalla tenda..
@crug80: aspetto queste feste e poi chiamerò direttamente la società che mi ha montato il tutto o direttamente il produttore. Ci avevo provato ma se ne parla ad anno nuovo...
grazie ad entrambi
